pubspec.yaml

```diff
@@ -2,15 +2,15 @@ name: very_good_infinite_list
 description: >-
   A Very Good Infinite List Widget created by Very Good Ventures.
   Comes in handy when making activity feeds, news feeds, etc.
-version: 0.1.1
+version: 1.0.0-nullsafety.0
```

Comment:
can me make this v0.2.0 instead since we are anticipating more changes to the API?